content/security.md

```diff
@@ -34,90 +35,91 @@ Without DNSSEC, a malicious hacker can point SimpleLogin MX record to their own
 
 CAA provides a way for domain holders to reduce that risk. Without CAA, someone could potentially obtain an unauthorized SSL certificate for SimpleLogin domains that could allow man-in-the-middle hacks.
 
-All SimpleLogin certificates are issued by [Letsencrypt](https://letsencrypt.org).
+All SimpleLogin certificates are issued by [Letsencrypt](https://letsencrypt.org). Witch is one of the most trusted CA's in the world.
 
-#### Hardenize reports
+#### Hardenize
 
-[Hardenize](https://www.hardenize.com) is a comprehensive security tool that provides assessment of web site network and security configuration.
+[Hardenize](https://www.hardenize.com) is a comprehensive security tool that provides assessments of website and network security configuration.
 
 Here are Hardenize reports for our domains:
 
-- [simplelogin.io](https://www.hardenize.com/report/simplelogin.io/1587326588)
-- [simplelogin.fr](https://www.hardenize.com/report/simplelogin.fr/1587547515)
-- [simplelogin.co](https://www.hardenize.com/report/simplelogin.co/1587326594)
-- [slmail.me](https://www.hardenize.com/report/slmail.me/1587417887#email_dmarc)
-
+- [simplelogin.io](https://www.hardenize.com/report/simplelogin.io)
+- [simplelogin.co](https://www.hardenize.com/report/simplelogin.co)
+- [simplelogin.fr](https://www.hardenize.com/report/simplelogin.fr)
+- [simplelogin.com](https://www.hardenize.com/report/simplelogin.com)
+- [slmail.me](https://www.hardenize.com/report/slmail.me)
```

Comment:
I think we can remove slmail.me, simplelogin.fr, simplelogin.co here as the official domains for SL is .io and .com

Reply:
Fair. Altho i would leave the .co in there, seen as you all operate your mail servers on that.